order.go

```diff
@@ -39,32 +39,125 @@ type OrderServiceOp struct {
 	client *Client
 }
 
+type orderStatus string
```

Comment:
`orderStatus` - I *think* lowercase types are not exported to outside the current package so that means no one would we able to use them - I don't think that's ok as you nee to be able to set it in the `OrderCountOptions` struct. 

Let me know if I got this wrong.

Reply:
Yes, `orderStatus` (and similar type definitions for consts, i.e.: `metafieldType) is not exported.  It does not need to be for a user to use the defined consts or to see the field in code-editor hints.  See screenshot of VSCode below.  As long as the parent struct's field (in the case of Metafields, the Type field) is exported, a user will be able to see it.

![image](https://github.com/bold-commerce/go-shopify/assets/6452664/b34914c0-0ab2-48fc-9eeb-61530dd18a53)

Reply:
Ah excellent, in that case we're good, thanks for explaining.